app/assets/stylesheets/_settings.scss

```diff
@@ -50,20 +50,21 @@ $global-font-size: 100%;
 $global-width: rem-calc(1200);
 $global-lineheight: 1.5;
 $foundation-palette: (
-  primary: #1779ba,
-  secondary: #767676,
+  primary: #265673,
+  secondary: #1a1a1a,
   success: #3adb76,
   warning: #ffae00,
   alert: #cc4b37,
 );
+$dark-blue: #265673;
 $light-gray: #e6e6e6;
 $medium-gray: #cacaca;
 $dark-gray: #8a8a8a;
-$black: #0a0a0a;
+$black: #000000;
 $white: #fefefe;
-$body-background: $white;
-$body-font-color: $black;
-$body-font-family: "Helvetica Neue", Helvetica, Roboto, Arial, sans-serif;
+$body-background: $black;
+$body-font-color: $dark-blue;
+$body-font-family: "Old Standard TT";
```

Comment:
As a best practice, your `font-family` property should always be a prioritized list, so as to provide fallback options when your first choice font is unavailable.